packages/editable-html/src/serialization.jsx

```diff
@@ -146,34 +146,9 @@ const marks = {
   }
 };
 
-const findPreviousText = el => {
```

Comment:
Please check my PR, which had changes there:
https://github.com/pie-framework/pie-lib/pull/243/files

It might fix that issue, anyhow removing all this functionality is not a good idea.

Reply:
Yes I have checked with your code, it fixes the issue I will close this PR. Thank you for the review